dashboard/modules/stats_collector/stats_collector_head.py

```diff
@@ -160,21 +172,40 @@ async def _update_actors(self):
         await aioredis_client.psubscribe(pattern)
         logger.info("Subscribed to %s", key)
 
+        def _process_actor_table_data(data):
+            actor_class = actor_classname_from_task_spec(
+                data.get("taskSpec", {}))
+            data["actorClass"] = actor_class
+
         # Get all actor info.
         while True:
             try:
                 logger.info("Getting all actor info from GCS.")
                 request = gcs_service_pb2.GetAllActorInfoRequest()
                 reply = await self._gcs_actor_info_stub.GetAllActorInfo(
-                    request, timeout=2)
+                    request, timeout=60)
```

Comment:
Load full actor info is very slow. But, I noticed that an optimization PR has been merged: https://github.com/ray-project/ray/pull/11067/

So, we can reduce the timeout here. The GCS serves RPC after all data has been loaded.  The `GetAllActorInfo` will return data from GCS cache, it will be fast.

Shall we set the timeout to 5? I think 2 seconds is too short for a real cluster.

Reply:
5 sounds good to me.